confignode/src/main/java/org/apache/iotdb/confignode/persistence/NodeInfo.java

```diff
@@ -242,6 +242,11 @@ public TSStatus updateConfigNodeList(ApplyConfigNodeReq applyConfigNodeReq) {
     TSStatus status = new TSStatus();
     configNodeInfoReadWriteLock.writeLock().lock();
     try {
+      if (nextNodeId.get() < applyConfigNodeReq.getConfigNodeLocation().getConfigNodeId()) {
+        // In this case, at least one ConfigNode is registered with the leader node,
+        // so the nextNodeID of the followers needs to be added
+        nextNodeId.getAndIncrement();
```

Comment:
nextDataNodeId.set(applyConfigNodeReq.getConfigNodeLocation().getConfigNodeId());    ?

Reply:
When there are multiple ConfigNode or DataNode registered concurrently, the applyReq with less id might arrive early.

Reply:
update